docs/becoming-a-maintainer/translations/es/problemas-y-solicitudes-de-extraccion.md

```diff
@@ -0,0 +1,181 @@
+# Cómo manejar problemas y solicitudes de extracción abiertas
+
+Una de las responsabilidades principales de un mantenedor de Open Source es priorizar los problemas abiertos y revisar las solicitudes de extracción. En este capítulo, hablaremos sobre cómo manejar mejor los problemas, las solicitudes de extracción y las vulnerabilidades de seguridad abiertas.
+
+## Priorización y gestión de problemas
+
+La priorización de problemas implica revisar una lista existente de problemas abiertos y priorizarlos en orden de importancia. Las siguientes secciones analizarán los tipos más comunes de problemas y cómo priorizarlos y responder mejor a los colaboradores.
+
+### Problemas etiquetados como errores
+
+Al priorizar problemas de errores, debes dedicar tiempo a reproducir el error para comprender el problema. Si no puedes reproducir el error, comunícate con otros miembros del equipo para ver si pueden reproducirlo. Si eres un mantenedor solitario, responde al autor original del problema para obtener más información y contexto.
+
+Luego, debes definir si el error se considera crítico, medio o pequeño.
+
+#### Errores críticos
+
+Cuando un problema abierto contiene un error que es un bloqueador importante y ha afectado la funcionalidad de toda la aplicación o sitio web, puedes considerarlo como alta prioridad o crítico. Este tipo de error debe corregirse de inmediato.
+
+Etiquetar estos problemas como `críticos` o `alta prioridad` es lo mejor para que el equipo sepa que deben abordarlos primero. Considera usar etiquetas en diferentes colores. Los colores de etiqueta rojo brillante o naranja son una buena opción porque indican el nivel de gravedad y son más fáciles de detectar en una lista de problemas.
+
+Para las correcciones críticas de errores, es mejor que los miembros del equipo central o los colaboradores habituales trabajen en estos problemas para garantizar que se hagan bien y con prontitud. Puedes usar una etiqueta como `core team work` para indicar que este problema solo está abierto para miembros selectos.
+
+#### Errores de pequeños a medianos
+
+Un error que no afecta la funcionalidad de tu aplicación no se considera crítico. Puedes categorizar los errores como de nivel medio cuando puedes corregirlos después de desplegar un lanzamiento próximo. Aquellos que puedas o no incluir en el desarrollo se consideran errores pequeños.
+
+Puedes abrir estos problemas para que cualquiera trabaje en ellos. Algunos de estos problemas pueden ser tan pequeños que serían una buena oportunidad para un nuevo colaborador. Etiqueta el problema con un `buen primer problema` o `solo para principiantes` en esta situación.
```

Comment:
`buen primer problema`
`solo para principiantes`

Should we keep these labels in english?

`good first issues` 
`first timers only`

I noticed in another file we kepted the labels `Needs triage` and `Bug` in english so figured we should stay consistent. :)